django_react_proj/users-fe/src/components/RealTime/RealTime.js

```diff
@@ -6,7 +6,7 @@ const Time = () => {
   const [time, setTime] = React.useState(0);
   const currentCallback = () => {
     const date = new Date();
-    const dateformat = format(date, 'HH:mm:ss');
+    const dateformat = format(date, 'HH:mm');
```

Comment:
If you changed the format to not display seconds an interval time should be also adjusted

Reply:
I didn't pay attention to it, but it has already been changed.